### PR TITLE
Website public beta 

### PR DESCRIPTION
**Pull request by timotheeguerin**
This is a feature branch containing the progress for the website

URL: https://tspwebsitepr.z22.web.core.windows.net/prs/2418/

### CODE REVIEW

File: eng/pipelines/pr.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ trigger:
 pr:
   - main
   - release/*
+  - feature/website-design
```
Comment (timotheeguerin) [inline eng/pipelines/pr.yml:6]: todo: remove before merging

File: packages/website/src/components/homepage/homepage.tsx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+            </Button>
+          </div>
+        </div>
+        <div className={style["hero-demo"]}>
+          <HeroTabs
+            tabs={[
+              {
+                value: "OpenAPI",
+                content: (
+                  <DotLottiePlayer src={useBaseUrl(`/lottie/openapi-hero.lottie`)} autoplay loop />
```
Comment (timotheeguerin) [inline packages/website/src/components/homepage/homepage.tsx:60]: File size is much better but now the decoding does take a little bit of time which happens everytime you switch tabs which is not optimal at all.

**Comment by mikekistler** [comment]
I think we need something on the Community page for how folks can ask questions, propose ideas, etc. I see that we've enabled discussions on the typespec repo -- is that were to point them? In any event, we should be clear about that.

**Comment by mikekistler** [comment]
There are several broken links on this page:

https://tspwebsitepr.z22.web.core.windows.net/prs/2595/openapi

Is there a way to integrate a link checker into the build to catch these without human testing?

**Comment by timotheeguerin** [comment]
> There are several broken links on this page:
> 
> https://tspwebsitepr.z22.web.core.windows.net/prs/2595/openapi
> 
> Is there a way to integrate a link checker into the build to catch these without human testing?

I'll take a look there is some issue with the preview website because azure storage website doesn't auto add `.html` when resolving the files so some links that might not work here will in the github.io version

File: .prettierignore
Comment (timotheeguerin) [inline .prettierignore:1]: Feedback from meeting:


"Description TBU" homepage update

- animation too much?
  - just type tsp and see openapi change


- "Describe APIs at scale" change wording 
 - subtitle is not bad?

Hero too much space? Know if you can scroll more? Show big down arrow?


- "Api-First" reword too "Api for developers"?


- Docs reorganize:
 - Guides first? progressive disclosure

- Playground default loaded example

File: packages/website/src/components/fluent-layout.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import BrowserOnly from "@docusaurus/BrowserOnly";
```
Comment (timotheeguerin) [inline packages/website/src/components/fluent-layout.tsx:1]: Combine with existing fluent layout

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**